dockstore-client/src/test/java/io/dockstore/client/cli/LaunchTestIT.java

```diff
@@ -1393,4 +1394,69 @@ public void cwlNullInputParameter() {
         // run simple echo null tool
         runTool(nullCWL, nullJSON, false);
     }
+
+    @Test
+    public void missingTestParameterFileWDL() {
```

Comment:
I forget, does WDL always require a test parameter file?

If not, can you add a test that successfully runs such a WDL. If yes, ignore this comment.